archive/glaciers_fig-1.csv

```diff
@@ -1,77 +1,75 @@
-"Figure 1. Average Cumulative Mass Balance of ""Reference"" Glaciers Worldwide, 1945-2014",,
-Source: EPA's Climate Change Indicators in the United States: www.epa.gov/climatechange/indicators,,
-"Data source: WGMS, 2015",,
-Web update: June 2015,,
+"Figure 1. Average Cumulative Mass Balance of ""Reference"" Glaciers Worldwide, 1956-2023",,
+Source: EPA's Climate Change Indicators in the United States: www.epa.gov/climate-indicators,,
+"Data source: WGMS, 2024",,
+Web update: June 2024,,
 Units: m of water equivalent; count,,
 ,,
 Year,Mean cumulative mass balance,Number of observations
-1945,0,
-1946,-1.13,1
-1947,-3.19,1
-1948,-3.19,1
-1949,-3.82,3
-1950,-4.887,3
-1951,-5.217,3
-1952,-5.707,3
-1953,-6.341,7
-1954,-6.825,6
-1955,-6.575,7
-1956,-6.814,7
-1957,-6.989,9
-1958,-7.693,9
-1959,-8.325,11
-1960,-8.688,14
-1961,-8.935,15
-1962,-9.109,20
-1963,-9.567,22
-1964,-9.699,22
-1965,-9.298,24
-1966,-9.436,27
-1967,-9.303,29
-1968,-9.219,31
-1969,-9.732,31
-1970,-10.128,32
-1971,-10.288,32
-1972,-10.441,32
-1973,-10.538,32
-1974,-10.613,32
-1975,-10.534,33
-1976,-10.633,35
-1977,-10.682,37
-1978,-10.754,37
-1979,-11.127,37
-1980,-11.318,36
-1981,-11.394,35
-1982,-11.849,36
-1983,-11.846,37
-1984,-11.902,37
-1985,-12.238,37
-1986,-12.782,37
-1987,-12.795,37
-1988,-13.26,37
-1989,-13.343,37
-1990,-13.687,37
-1991,-14.255,37
-1992,-14.501,36
-1993,-14.695,37
-1994,-15.276,37
-1995,-15.486,37
-1996,-15.89,37
-1997,-16.487,37
-1998,-17.31,37
-1999,-17.697,37
-2000,-17.727,37
-2001,-18.032,37
-2002,-18.726,37
-2003,-19.984,37
-2004,-20.703,37
-2005,-21.405,37
-2006,-22.595,37
-2007,-23.255,37
-2008,-23.776,37
-2009,-24.459,37
-2010,-25.158,37
-2011,-26.294,37
-2012,-26.93,36
-2013,-27.817,31
-2014,-28.652,24
+1956,0,

```

Comment:
@gradedSystem do you know why we have data starting from 1956 vs in the old file we had 1945?

Reply:
In the official website it is said like this:
> Figure 1 shows trends in mass balance for a set of 61 reference glaciers around the world that have been measured consistently since the 1970s, including several that have been measured since the 1950s. Data from these reference glaciers have been averaged together to depict changes over time
@anuveyatsu

Reply:
So I think those from 1945 are merged until 1956
@anuveyatsu